SwahiliBox/swahilibox-events-api
# Implement skills endpoint

#### What does this PR do?
implements the skills endpoints.

#### Description of Task to be completed?
Implement skills resource functions, service functions, and routes.
#### How should this be manually tested?
Pull these changes, delete your tables and run migrations again. (These changes modify migration files directly as there is no production data to be lost.)
You should be able to perform CRUD operations on the `/skills` endpoint.
All new and existing tests should pass.

#### Any background context you want to provide?
This PR resolves issue #49 
#### What are the relevant tasks or issues?
#49
